cerberus/kubernetes/client.py

```diff
@@ -175,20 +167,22 @@ def process_nodes(watch_nodes, iteration, iter_track_time):
 def namespace_sleep_tracker(namespace, pods_tracker):
     crashed_restarted_pods = defaultdict(list)
     all_pod_info = get_all_pod_info(namespace)
-    if all_pod_info != "" and len(all_pod_info) > 0:
-        for pod_info in all_pod_info["items"]:
-            pod = pod_info["metadata"]["name"]
-            pod_status = pod_info["status"]
-            pod_status_phase = pod_status["phase"]
+    if all_pod_info != "":
```

Comment:
This type of object (V1Pod, from kubernetes)  doesn't have a length, that's why I had to take it out. I can try to add in a check to make sure it is also not None before going into that if statement.

Reply:
In case when 'No resources is found', items is  set to empty list [ ] in the returned object. so checking  `len(all_pod_info.items)` might also help.

`
{'api_version': 'v1',
 'items': [],
 'kind': 'PodList',
 'metadata': {'_continue': None,
              'remaining_item_count': None,
              'resource_version': '1575517',
              'self_link': None}}
`

Reply:
If the len(all_pod_info.items) is 0 it wouldn't go into that for loop though so do we need that check?

Reply:
if  its empty list , it just wouldn’t go into the for loop. I guess current flow work.
+1